example/example.cfg.example

```diff
@@ -27,6 +28,12 @@ MULTIAUTH_AUTH_PROVIDERS = {
     'local': {
         'type': 'example_local',
         'title': 'Local Accounts'
+    },
+    'my_shibboleth': {
+        'type': 'shibboleth',
+        'title': 'Shibboleth',
+        'callback_uri': '/shibboleth/sso',
```

Comment:
It's something you have to configure manually in the Shibboleth config anyway, so I'm not against it.